sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -994,18 +999,21 @@ class SGDClassifier(BaseSGDClassifier):
         important to get the prediction exactly right.
         For epsilon-insensitive, any differences between the current prediction
         and the correct label are ignored if they are less than this threshold.
+        Values should be in the range `[0.0, inf)`.
 
     n_jobs : int, default=None
         The number of CPUs to use to do the OVA (One Versus All, for
         multi-class problems) computation.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
+        Values should be in the range `[-1, inf)`.
 
     random_state : int, RandomState instance, default=None
         Used for shuffling the data, when ``shuffle`` is set to ``True``.
         Pass an int for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
+        Values should be in the range `[0, inf)`.
```

Comment:
```suggestion
        Integer values should be in the range `[0, 2**32 - 1]`.
```

Reply:
Maybe this range info could be added to the Glossary entry for `random_state` instead of putting it here. Or it could be added in both locations.